### ISSUE TITLE
Retention Alignment 

### ISSUE THREAD

**Issue by holandir** [jorainer/xcmsTutorials#21]
Hello,

I have a problem with the retention time alignment of my LC-MS samples.
The "Obiwarp" method works well, but when I try to do the "peakGroups" method, I get the following error:

Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘groupChromPeaks’ for signature ‘"MsExperiment", "PeakDensityParam"’. 

This is the line of code I used (mse2 being my MsExperiment data set)

pdp <- PeakDensityParam(sampleGroups = sampleData(mse2)$sample_group, bw = 1.8,
                        minFraction = 1, binSize = 0.01, ppm = 10)
mse2 <- groupChromPeaks(mse2, pdp)

How can I solve this problem?
I suspect a version-issue.

**Comment by jorainer** [comment]
yep, this is for sure a version issue. The functionality to use `xcms` with `MsExperiment` was added in Bioconductor release 3.18 - so you need at least `xcms` version 4.0.0 for that. To install these versions you would need a recent R (4.3.2) and then install the respective packages using

```r
install.packages("BiocManager")
BiocManager::install(c("xcms", "MsExperiment"))
```

There is also some code here and there in this tutorial that is (at present) only available in the current developmental packages (`xcms` >= 4.1.0). These can at present only be installed through github - but will become the *official* version in Bioconductor release 3.19 which should be released in April.
